pkg/serviceimport/map.go

```diff
@@ -36,9 +36,7 @@ type Map struct {
 	sync.RWMutex
 }
 
-func (m *Map) selectIP(si *serviceInfo, checkCluster func(string) bool) (string, bool) {
-	queue, counter := si.clustersQueue, &si.rrCount
-
+func (m *Map) selectIP(queue []clusterInfo, counter *uint64, checkCluster func(string) bool) (string, bool) {
 	if queue == nil {
 		return "", false
 	}
```

Comment:
We don't need this anymore.  Nor do we need to return the "found" boolean as the caller has already determined that.

Reply:
done